onnx/defs/gen_doc.py

```diff
@@ -20,7 +20,7 @@
 
 SNIPPETS = collect_snippets()
 SAMPLE_IMPLEMENTATIONS = collect_sample_implementations()
-ONNX_ML = bool(os.getenv('ONNX_ML') == '1')
+ONNX_ML = not bool(os.getenv('ONNX_ML') == '0')
```

Comment:
The file permission has been changed.

Reply:
Done. I guess you'd like to keep 644 because users need to specify python/python3 explicitly? If this is the case, how about removing shebangs from 644 files?